src/foam/nanos/notification/email/EmailTemplateSourceEnum.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * @license
+ * Copyright 2021 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+ foam.ENUM({
+  package: 'foam.nanos.notification.email',
+  name: 'EmailTemplateSourceEnum',
+  documentation: `This is used to define a source of email's generation type.
+    A source of email's generation is where sending email function gets called
+    and It can be the below.
+  `,
+
+  values: [
+    {
+      name: 'UNDEFINED',
+      label: 'Undefined',
+      ordinal: 0
+    },
+    {
+      name: 'RULE',
+      label: 'Rule',
```

Comment:
Are the labels required? Won't FOAM set to the same values anyway?

Reply:
Also, why is this name all upper case?